### DISCUSSION TITLE
JayzTwoCents shows this app

### DISCUSSION THREAD

**Discussion by Haplo66** [LibreHardwareMonitor/LibreHardwareMonitor#1391] [General]
https://youtu.be/sX2ThUWNXio?si=POGrn8N4an4e8cVD

**Comment by Kaladn** [comment]
I've seen it and it was excellent. He referred to it in a recent video. I've downloaded the it; it's comprehensive, covering a wide array of sensors and such. However, it's extremely bright. I am looking for a way to change the colors.

**Reply by RadekJurzysta** [reply]
Jay's first request was to add dark mode to the app :)
